Example-Cobol-AirportService/cobol/airport.cbl

```diff
@@ -1353,7 +1353,7 @@
            *> Establish addressability to the airport data structure
            Set address of resp-body-data to resp-body-data-ptr.
 
-           *> Perform a sanity check on the airport-info data.
+           *> Perform a eyecatcher check on the airport-info data.
```

Comment:
should the a be an an? 'Peform an eyecatcher check?'

Reply:
fair...